.github/workflows/velox.yml

```diff
@@ -0,0 +1,76 @@
+name: Velox Nightly Build and Test
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  container:
+    runs-on: ubuntu-latest
+    container: prestocpp/velox-velox-torcharrow:kpai-20220301
+    steps:
+      - name: Print CPU info
+        run: cat /proc/cpuinfo
+
+      - name: Setup Python environment
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
```

Comment:
I see Python environment is set up by conda -- do we need to set up Python environment through GitHub Actions?